python-checks/src/test/resources/checks/robustCipherAlgorithm.py

```diff
@@ -98,3 +114,19 @@ def pycryptoCompliant():
   from Crypto.Cipher import *
   aes1 = AES.new('This is a key123', AES.MODE_CBC, 'This is an IV456') # Compliant
 
+def pysslCompliant():
```

Comment:
I see it was not done before but CamelCase is not really Pythonic. I don't think it needs to be changed here, but just noting it.

Reply:
Ok, thank you. I think it's maybe worth changing these function names.

Reply:
Fixed in 03af7494fb21355e655e0b166750628ac16418f1.